README.md

```diff
@@ -78,17 +78,14 @@ the system will ask the service desk if any agents are available. If no response
 
 To set up your development environment, first fork this repository. 
 
-Copy `.env-sample` to `.env` and change the `SERVICE_DESK_CLASS` variable to run the target service desk implementation. If you do not provide any value, it will run the Mock service desk that we provide by default.
+Instructions for setting up a development environment for each of the service desk reference example integrations can be found in the following paths:
+- [Genesys Cloud](./src/genesys/webChat/server)  
+- [Twilio Flex](./src/flex/webChat/server) 
+- [NICE inContact](./src/incontact/webChat/server)
 
-If you are running an integration which needs middleware, then `src/middleware` folder has a corresponding folders for middlewares which has instructions on how to run the middleware.
+To run the default example, change to the `src/example/webChat/client` directory and run `npm install`. Then run `npm run dev` to launch the example service desk webchat client.
 
-Run `npm install` 
```

Comment:
Lets bring all this deleted stuff back and add it to the README.md of the example folder.